Xamarin.Forms.Platform.Android/Renderers/OpenGLViewRenderer.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Xamarin.Forms.Platform.Android
 {
-	internal class OpenGLViewRenderer : ViewRenderer<OpenGLView, GLSurfaceView>
+	public class OpenGLViewRenderer : ViewRenderer<OpenGLView, GLSurfaceView>
```

Comment:
does this change make sense?

Reply:
To explicitly register the renderer it needs to be public.